### PR TITLE
feature/AT-RSC-UI- Added Sprint Views and Sections

### PR DESCRIPTION
**Pull request by Normaillescas29**
Added Sprint Views and Sections

### CODE REVIEW

**Review by irvingesp2190 — APPROVED** [review]

File: src/controllers/at-resources.controller.js [hunk 1]
```diff
@@ -6,4 +6,35 @@ rscCtrl.dashboard = async(req, res) => {
...
+//-----------Sprint-----------//
+
+// Render Sprint
+rscCtrl.renderSprint = (req, res) => {
+    res.render("sprint");
+};
+
+// Render addSprint
+rscCtrl.renderAddSprintForm = (req, res) =>{
+res.render("/sprint/add");
```
Comment (at-internship) [inline src/controllers/at-resources.controller.js:20]: Please indent code

File: src/views/partials/menu-at-resources.hbs [hunk 2]
```diff
@@ -0,0 +1,132 @@
+<!-- MENU -->
+<ul class="nav flex-column">
+    <li class="nav-item">
+        <a class="nav-link" href="/at-resources/dashboard">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shold be `<a class="nav-link" href="/dashboard">`

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/controllers/at-resources.controller.js (hunk 1 above)
Comment (Normaillescas29) [reply src/controllers/at-resources.controller.js:20]: Do we need to create a form to add a new Sprint?

Comment (at-internship) [reply src/controllers/at-resources.controller.js:20]: at this point, only on "admin" section

Comment (Normaillescas29) [reply src/controllers/at-resources.controller.js:20]: Then, do we need to remove or comment this method for this moment ?

**Comment by at-internship** [comment]
This pull must be merged into develop not feature/AT-RSC-UI_Migration

File: src/controllers/at-resources.controller.js (hunk 1 above)
Comment (at-internship) [reply src/controllers/at-resources.controller.js:20]: you can leave it

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**